app/routes.py

```diff
@@ -1 +1,218 @@
-from flask import Blueprint
\ No newline at end of file
+import os
+import requests
+from datetime import datetime
+from flask import Flask, Blueprint, jsonify, request, abort, make_response
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from .routes_helper import get_one_obj_or_abort
+
+
+task_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
+
+# --------------------------  Task Endpoints --------------------------
+@task_bp.route("", methods=["POST"])
+def create_a_task():
+    request_body = request.get_json()
+    new_task = Task.from_dict(request_body)
+
+    # new_task = Task(title=request_body.get("title", None),
+    #                 description=request_body.get("description", None))
+    if new_task.title is None or new_task.description is None:
+        return jsonify({"details": "Invalid data"}),400
```

Comment:
Good that you're doing data validation here, but maybe let the user know what data was invalid as well.